src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx

```diff
@@ -414,13 +423,41 @@ export const EditVideoForm: React.FC<EditVideoFormProps> = ({
           </FormField>
           {isEdit && (
             <DeleteVideoContainer>
-              <DeleteVideoButton size="large" variant="tertiary" textColorVariant="error" onClick={handleDeleteVideo}>
+              <DeleteVideoButton
+                size="large"
+                variant="tertiary"
+                textColorVariant="error"
+                onClick={() => {
+                  handleDeleteVideoClick(selectedVideoTab?.id)
+                }}
+              >
                 Delete video
               </DeleteVideoButton>
             </DeleteVideoContainer>
           )}
         </InputsContainer>
       </FormWrapper>
+      <MessageDialog
+        title="Do you want to remove this video from your videos?"
+        exitButton={false}
+        description="Video will be removed permanently and all its data will be lost. Joystream studio do not keep any of your data after you remove your video."
+        showDialog={!!videoIdToDelete}
+        onSecondaryButtonClick={handleCancel}
+        onPrimaryButtonClick={handleConfirmDeleteVideo}
+        error
+        variant="warning"
+        primaryButtonText="Delete video"
+        secondaryButtonText="Cancel"
+      />
+      <TransactionDialog
+        status={deleteTransactionStatus}
+        successTitle={'Video successfully deleted!'}
+        successDescription={'Your video was deleted from the blockchain.'}
```

Comment:
```suggestion
        successDescription="Your video was marked as deleted and it will no longer show up on Joystream."
```

FYI, the reason for this copy change is that once something is saved on the blockchain, it cannot really be deleted ever, so I guess it's good to not give false understanding to the user